addons/popochiu/editor/popups/create_prop/create_prop.gd

```diff
@@ -93,10 +93,13 @@ func _create() -> void:
 	
 	# ▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓▓
 	# Add the prop to its room
-	_room.get_node('Props').add_child(prop)
+	# Instancing the created .tscn file fixes #58
+	var prop_instance: PopochiuProp = load(_new_prop_path + '.tscn').instantiate()
 	
-	prop.owner = _room
-	prop.position = Vector2(
+	_room.get_node('Props').add_child(prop_instance)
```

Comment:
Not blocking for this PR but maybe this logic belongs to the Room? So it can expose the logic to add a new prop to itself?

Reply:
Since the creation of props implies the creation of folders and files in the system, I took the decision to separate the functionality from the Room (which should handle the logic related to in-game actions). But I'm open to discuss this, of course.

Reply:
Well, it makes perfect sense actually. Then, I would move all the creation/deletion logic to a utility class (a factory or repository). That way we can refer to that class everywhere we are trying to automatize those operations since they can become pretty complex and require context validation.

What do you think?

Reply:
Yes. Indeed, a thing I want to refactor is the creation popups. At the moment some of them perform similar operations (in terms of creating folders and files) but the logic is copied here and there. Now that you'll need some of those things for the importer, having that logic in a separate script will make our lives easier, and the project better structured.